legend-depot-artifacts-refresh/src/main/java/org/finos/legend/depot/store/artifacts/services/ProjectVersionRefreshHandler.java

```diff
@@ -82,10 +83,11 @@ public final class ProjectVersionRefreshHandler implements NotificationEventHand
     private final List<String> projectPropertiesInScope;
     private final Queue workQueue;
     private final DependencyManager dependencyManager;
+    private final int noOfSnapshotVersions;
 
 
     @Inject
-    public ProjectVersionRefreshHandler(ManageProjectsService projects, RepositoryServices repositoryServices, Queue workQueue, RefreshStatusStore store, ArtifactsFilesStore artifacts, IncludeProjectPropertiesConfiguration includePropertyConfig, DependencyManager dependencyManager)
+    public ProjectVersionRefreshHandler(ManageProjectsService projects, RepositoryServices repositoryServices, Queue workQueue, RefreshStatusStore store, ArtifactsFilesStore artifacts, IncludeProjectPropertiesConfiguration includePropertyConfig, DependencyManager dependencyManager, RetentionPolicyConfiguration retentionPolicyConfiguration)
```

Comment:
please dont pass the entiry configuration just the specific item u need from it..
the place  where u have put the definition for the config is not correct. move it o where it belongs and expose wht u need from it

Reply:
now I inject only the snapshot information